README.md

```diff
@@ -56,7 +56,7 @@ which are part of the [SUSE Linux Enterprise Server for SAP Applications](https:
 
 These could in theory also be installed and configured on other distributions providing the same functionalities, but this use case is not within the scope of the active development.
 
-In addition to that, _Trento Agent_ also requires the [Prometheus node_exporter component](https://github.com/prometheus/node_exporter) to be running to collect host information for the monitoring functionality.
+In addition to that, _Trento Agent_ also wants the [Prometheus node_exporter component](https://github.com/prometheus/node_exporter) to be running to collect host information for the monitoring functionality.
```

Comment:
I would avoid specific systemd lingo here and better articulate that the integration is optional.
```suggestion
In addition to that, _Trento Agent_ can also optionally integrate with the [Prometheus node_exporter component](https://github.com/prometheus/node_exporter) to collect host information for the monitoring functionality.
```